views/oleville/index.js

```diff
@@ -85,6 +85,9 @@ export default class OlevilleView extends React.Component {
         return 'http://oleville.com/wp-content/uploads/2015/12/Oleville-Logo.png'
       }
     }
+    else {
+        return 'http://oleville.com/wp-content/uploads/2015/12/Oleville-Logo.png'
+    }
```

Comment:
What about _only_ having this case, and removing the inner else-statement?

Actually: Given https://github.com/StoDevX/AAO-React-Native/blob/6f7b172ddde96b1960077e0f8d8ffc22e47372e3/views/oleville/index.js#L74-L91, 

```diff
  async getImageUrl(id: number) {
    if (id) {
      let response = await fetch(`http://oleville.com/wp-json/wp/v2/media/${id}`)
      let responseJson = await response.json()
      this.setState({loaded: true})

      if (responseJson.media_details.sizes.medium.source_url) {
        // if there is a featured image, use that url
        return responseJson.media_details.sizes.medium.source_url
+      }
-      } else {
-        // otherwise default to the oleville logo
-        return 'http://oleville.com/wp-content/uploads/2015/12/Oleville-Logo.png'
-      }
     }
-    else {
-        return 'http://oleville.com/wp-content/uploads/2015/12/Oleville-Logo.png'
-    }
+    return 'http://oleville.com/wp-content/uploads/2015/12/Oleville-Logo.png'
  }
```

That is, removing _both_ else-blocks entirely and just letting the function fall back to a single value?